make_results_table.py

```diff
@@ -0,0 +1,19 @@
+import jinja2
+
+from libraries_io_scraper.models import Dependency
+
+
+
+def make_results_table(dependencies:  dict[str, list[Dependency]]):
+    output = populate_jinja_template(dependencies=dependencies)
+    write_template_to_file(output=output)
+
+def populate_jinja_template(dependencies: dict[str, list[Dependency]]) -> str:
+    environment = jinja2.Environment(loader=jinja2.FileSystemLoader("libraries_io_scraper/results_table"))
+    template = environment.get_template("markdown_table_template.j2")
+    output = template.render(dependencies=dependencies)
+    return output
+
+def write_template_to_file(output: str):
+    with open('dependency_ratings.md', 'w') as f:
+        f.write(output)
```

Comment:
It's because I was naive about the use of `Dependency`; `.name` immediately gets mangled into the `urlib.parse.quote` version of itself so that it can be passed nicely to the libraries.io api. I'll fix it in `main` and drop you a line when it's done.

Reply:
All fixed, and tests passing.